### PR TITLE
Fixes spelling H3-related code

### PR DESCRIPTION
**Pull request by randall**

### CODE REVIEW

File: iocore/net/quic/QUICAckFrameCreator.h [hunk 1]
```diff
@@ -69,7 +69,7 @@ class QUICAckFrameManager : public QUICFrameGenerator
     bool _available                         = false; // packet_number has data to sent
     bool _should_send                       = false; // ack frame should be sent immediately
     bool _has_new_data                      = false; // new data after last sent
-    uint32_t _ack_eliciting_count           = 0;     // every two ack-eliciting packet should send ack immediatly
+    uint32_t _ack_eliciting_count           = 0;     // every two ack-eliciting packet should send ack imeediately
```
Comment (bneradt) [inline iocore/net/quic/QUICAckFrameCreator.h:72]: imeediately

File: proxy/http3/Http3App.h [hunk 2]
```diff
@@ -106,6 +106,6 @@ class Http3SettingsFramer : public Http3FrameGenerator
 
 private:
   NetVConnectionContext_t _context;
-  bool _is_done = false; ///< Becarefull when set FIN flag on CONTROL stream. Maybe never?
+  bool _is_done = false; ///< be careful when set FIN flag on CONTROL stream. Maybe never?
```
Comment (bneradt) [inline proxy/http3/Http3App.h:109]: Probably good to capitalize that first 'b' here to be consistent with the next comment.

File: iocore/net/quic/QUICAckFrameCreator.h (hunk 1 above)
Comment (randall) [reply iocore/net/quic/QUICAckFrameCreator.h]: that's embarrassing! :) Thanks for the catch!

**Comment by zwoop** [comment]
Cherry-picked to v9.0.x branch.

**Pull request merged**